guides/release/typescript/application-development/converting-an-app.md

```diff
@@ -83,23 +116,29 @@ In general, we recommend migrating to Octane idioms before, or in conjunction wi
 
 ## ember-cli-typescript
 
-If you're migrating from `ember-cli-typescript`, particularly an older version, to Ember's out-of-the-box TypeScript support, you may also need to update your `tsconfig.json`. Current versions of `ember-cli-typescript` generate the correct config at installation. You do _not_ need `ember-cli-typescript` installed for new apps or addons.
+The `ember-cli-typescript` package was used to add TypeScript support to Ember apps before Ember's native TypeScript support was available. You do _not_ need `ember-cli-typescript` installed for new apps or addons.
+
+If you're migrating from `ember-cli-typescript` to Ember's native TypeScript support, most of your existing configuration will still be relevant. Just read through the steps of this guide and ensure that your config matches the expected config as described above.
 
 <!-- Internal links -->
 
+[ED-ts-guides]: ../../core-concepts/ember-data/#toc_adding-emberdata-types-to-an-existing-typescript-app
 [getting started]: ../../getting-started/
 [legacy]: ../../additional-resources/legacy/
+[packages]: ../../getting-started/#toc_packages-to-support-typescript
 [strictness]: ../../additional-resources/faq/#toc_strictness
 
 <!-- External links -->
 
 [allowJs]: https://www.typescriptlang.org/tsconfig/#allowJs
 [any]: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#any
 [dts]: https://www.typescriptlang.org/docs/handbook/declaration-files/introduction.html
+[environment.d.ts]: https://github.com/ember-cli/editor-output/blob/stackblitz-app-output-typescript/app/config/environment.d.ts
 [eslintrc]: https://github.com/ember-cli/editor-output/blob/stackblitz-app-output-typescript/.eslintrc.js
```

Comment:
this file doesn't exist anymore, it's now mjs

it's now here: https://github.com/ember-cli/editor-output/blob/stackblitz-app-output-typescript/eslint.config.mjs

```suggestion
[eslintrc]: https://github.com/ember-cli/editor-output/blob/stackblitz-app-output-typescript/eslint.config.mjs
```